digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -160,9 +160,13 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     val test = parsedBody.extract[newRoadAddressProject]
     val user = userProvider.getCurrentUser()
     val formatter = DateTimeFormat.forPattern("dd.MM.yyyy")
-    val roadAddressProject  = RoadAddressProject( 0, 1, test.name, user.username, "-", formatter.parseDateTime(test.startDate), DateTime.now(), test.additionalInfo, test.roadNumber, test.startPart, test.endPart)
+    val roadAddressProject  = RoadAddressProject( 0, 1, test.name, user.username, DateTime.now(), "-", formatter.parseDateTime(test.startDate), DateTime.now(), test.additionalInfo, test.roadNumber, test.startPart, test.endPart)
```

Comment:
It is taking the next id sequence inside roadAddressService.saveRoadAddressProject. The value is 0 by now because we are not updating yet the project from form on this US. We will later have id and status values directly from form.